### ISSUE TITLE
RobotContainer::movePivotSafe doesn't do anything

### ISSUE THREAD

**Issue by drsandytoes** [FRCTeam1967/2024Crescendo#10]
https://github.com/FRCTeam1967/2024Crescendo/blob/cdccfde96d5f128416030d6f4ce314171f30ccc9/2024Crescendo/src/main/java/frc/robot/RobotContainer.java#L123

This doesn't do anything, AFAICT. It creates a new MovePivot command, but it's never scheduled so nothing is going to happen with it.

**Comment by sanatiwary** [comment]
We use this in auto init, it was to try and combat the intake moving down randomly. We've commented it out now because our pivot isn't moving down at all but it's still in latest push's auto init uncommented

**Comment by drsandytoes** [comment]
I don't think it was causing any harm. It just wasn't doing anything to help either.
